content/learn/contribute/helping-out/opening-pull-requests.md

```diff
@@ -10,16 +10,24 @@ weight = 3
 Most changes don't require much "process". If your change is relatively straightforward, just do the following:
 
 1. A community member (that's you!) creates one of the following:
-    * [GitHub Discussions]: An informal discussion with the community. This is the place to start if you want to propose a feature or specific implementation.
-    * [Issue](https://github.com/bevyengine/bevy/issues): A formal way for us to track a bug or feature. Please look for duplicates before opening a new issue and consider starting with a Discussion.
-    * [Pull Request](https://github.com/bevyengine/bevy/pulls) (or PR for short): A request to merge code changes. This starts our "review process". You are welcome to start with a pull request, but consider starting with an Issue or Discussion for larger changes (or if you aren't certain about a design). We don't want anyone to waste their time on code that didn't have a chance to be merged! But conversely, sometimes PRs are the most efficient way to propose a change. Use your own judgment here.
-2. Other community members review and comment in an ad-hoc fashion. Active Subject Matter Experts may be pulled into a thread using `@mentions`. If your PR has been quiet for a while and is ready for review, feel free to leave a message to "bump" the thread, or bring it up on [Discord] in an appropriate engine development channel.
+   * [GitHub Discussion]: An informal discussion with the community.
+
+      This is the place to start if you want to propose a feature or specific implementation.
+   * [Issue](https://github.com/bevyengine/bevy/issues): A formal way for us to track a bug or feature.
+
+      Please look for duplicates before opening a new issue and consider starting with a Discussion.
+   * [Pull Request](https://github.com/bevyengine/bevy/pulls) (or PR for short): A request to merge code changes.
```

Comment:
I find this (and the other similar cases) a little harder to read in markdown form: there's an empty line between the previous bullet point and the last paragraph, but not between the last paragraph and the new bullet point even though they're not related.

Would it render the same by adding an empty line between those two?

Reply:
Unfortunately, no. It just creates a continuous paragraph from the bullet point.

<img width="884" alt="image" src="https://github.com/user-attachments/assets/aaea9980-dd13-4b7f-8c53-bb480f3be1c5" />

Reply:
Read your comment wrong at first, seems like zola/markdown are okay with that.